assets/prototype/domain/eventEditor/tickets/ticketPriceCalculator/formDecorators/usePriceTypeDecorator.js

```diff
@@ -0,0 +1,67 @@
+import { allPass, map, propEq, when } from 'ramda';
+import { isEqual, getFromFormData } from './utilities';
+import useTicketPriceCalculator from '../hooks/useTicketPriceCalculator';
+import usePriceTypes from '../../../data/queries/priceTypes/usePriceTypes';
+import { getPriceType } from '../../../../shared/predicates/prices/selectionPredicates';
+import { getDefaultPriceType } from '../../../../shared/predicates/priceTypes/selectionPredicates';
+
+const usePriceTypeDecorator = () => {
+	const priceTypes = usePriceTypes();
+	const getPriceTypeForPrice = getPriceType(priceTypes);
```

Comment:
I had to think twice what is the difference between `usePriceTypes` and `getPriceType` :smiley: